src/components/PaginationV2/PaginationV2.js

```diff
@@ -188,16 +188,17 @@ export default class PaginationV2 extends Component {
           <button
             className="bx--pagination__button bx--pagination__button--backward"
             onClick={this.decrementPage}
-            disabled={this.props.disabled || statePage === 1}>
+            disabled={this.props.disabled || statePage === 1}
+            style={
+              pageInputDisabled ? { borderRight: 0, marginRight: '1px' } : null
```

Comment:
Thank you for fixing this @tw15egan! Any way to add a custom style to avoid in-line?

Reply:
Would you prefer we add a class here (`bx--pagination__button--no-index`)? And then have the style come from a change in `carbon-components`?

Reply:
@tw15egan Thank you for double-checking! Yes I think this is the right way to move forward.